tests/outofband.test.ts

```diff
@@ -91,18 +96,19 @@ describe('OutOfBandController', () => {
       const response = await request(app).post('/oob/create-invitation')
 
       expect(response.statusCode).to.be.equal(200)
-      expect(response.body).to.deep.equal(objectToJson({
-        invitationUrl: outOfBandRecord.outOfBandInvitation.toUrl({
-          domain: bobAgent.config.endpoints[0],
-        }),
-        invitation: outOfBandRecord.outOfBandInvitation.toJSON({
-          useLegacyDidSovPrefix: bobAgent.config.useLegacyDidSovPrefix,
-        }),
-        outOfBandRecord: outOfBandRecord.toJSON(),
-      }))
-      createInvitationStub.restore()
+      expect(response.body).to.deep.include(
+        // remove properties with value undefined
```

Comment:
Keep this commented line?

Reply:
again yes, the stringify, parse combo below is odd and this explains the intent